src/python/pybind11/CMakeLists.txt

```diff
@@ -31,7 +33,7 @@ if (HAS_PYTHON)
     SET(PKGDIR "site-packages")
   endif(${LINUX_FLAVOR} MATCHES "^(ubuntu|debian)")
 
-  find_package(pybind11 2.6.0 REQUIRED PATHS "/usr/local/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11" "/usr/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11")
+  find_package(pybind11 2.10.4 REQUIRED PATHS "/usr/local/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11" "/usr/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11")
```

Comment:
Requirement for Version 2.10.4 is causing build failure. We should keep the required version at 2.6.0 as that what passes build today.

> [2023-06-09T18:42:13.630Z] CMake Error at python/pybind11/CMakeLists.txt:75 (find_package):
> [2023-06-09T18:42:13.630Z]   Could not find a package configuration file provided by "pybind11"
> [2023-06-09T18:42:13.630Z]   (requested version 2.10.4) with any of the following names:
> [2023-06-09T18:42:13.630Z] 
> [2023-06-09T18:42:13.630Z]     pybind11Config.cmake
> [2023-06-09T18:42:13.630Z]     pybind11-config.cmake
> [2023-06-09T18:42:13.630Z] 
> [2023-06-09T18:42:13.630Z]   Add the installation prefix of "pybind11" to CMAKE_PREFIX_PATH or set
> [2023-06-09T18:42:13.630Z]   "pybind11_DIR" to a directory containing one of the above files.  If
> [2023-06-09T18:42:13.630Z]   "pybind11" provides a separate development package or SDK, be sure it has
> [2023-06-09T18:42:13.630Z]   been installed.